### DISCUSSION TITLE
[Bug] Auth Setup background image wrongful path

### DISCUSSION THREAD

**Discussion by roenfeldt** [thedevdojo/wave#162] [General]
There is a problem with the background image for the auth views. More specifically, after authentication as the admin, when navigating to [https://wave.devdojo.com/auth/setup/appearance?tab=background](https://wave.devdojo.com/auth/setup/appearance?tab=background), the current background image path is wrong, resulting in a 404 exception in the browser's dev tools.
![bg-wrong-path-bug](https://github.com/user-attachments/assets/5dbd1eb6-f6fe-47b9-91e4-640d73fc0f42)

Considering the fact that the Auth Setup resources are located in the `/vendor` dir, and that there are no publishable resources for them, what's the recommended approach in order to fix this problem?

Thanks!

**Comment by bobbyiliev** [comment]
Hey!

Ah yes, I was also able to reproduce this. I've submitted a PR with a fix for this here: https://github.com/thedevdojo/auth/pull/140

This functionality is handled by the auth package itself.

Thank you for reporting this bug!

\- Bobby
